packages/docs/site/i18n/ja/docusaurus-plugin-content-docs/current/blueprints/tutorial/01-what-are-blueprints-what-you-can-do-with-them.md

```diff
@@ -0,0 +1,126 @@
+---
+title: ブループリントとは何ですか?
+slug: /blueprints/tutorial/what-are-blueprints-what-you-can-do-with-them
+description: ブループリントを使い始める
+---
+
+# ブループリントとは何ですか? ブループリントで何ができますか?
+
+<!--
+# What are Blueprints, and what can you do with them?
+ -->
+
+WordPress Playground を使用すると、プラグイン、テーマ、コンテンツ (投稿、ページ、分類、コメント)、設定 (サイト名、ユーザー、パーマリンクなど) などを含む Web サイト全体を作成できます。製品が揃った WooCommerce ストア、記事が掲載された雑誌、複数のユーザーがいる企業ブログなどを生成できます。
+
+<!--
+With WordPress Playground you can create a whole website, including plugins, themes, content (posts, pages, taxonomy, and comments), settings (site name, users, permalinks, and more), etc. They allow you to generate a WooCommerce store complete with products, a magazine populated with articles, a corporate blog with multiple users, and more.
+ -->
+
+ブループリントは、Playground インスタンスを構成するために使用できる `JSON` ファイルです。
+
+<!--
+Blueprints are `JSON` files that you can use to configure Playground instances.
+ -->
+
+ブループリントは、ファイルシステムやデータベースの操作といった高度なユースケースをサポートし、作成するインスタンスをきめ細かく制御できます。WordPress テストチームは、[6.5 ベータ版リリースサイクル](https://wordpress.org/news/2024/03/wordpress-6-5-release-candidate-2/)で Playground を使用し、最新バージョン、複数のテストプラグイン、ダミーデータを読み込むブループリントを作成しています。
+
+<!--
+Blueprints support advanced use cases, like file system and database manipulation, and give you fine-grained control over the instance you create. The WordPress Test Team has been using Playground in [the 6.5 beta release cycle](https://wordpress.org/news/2024/03/wordpress-6-5-release-candidate-2/), creating a Blueprint that loads the latest version, several testing plugins, and dummy data.
+ -->
+
+## 簡単な例
+
+<!--
+## A simple example
+ -->
+
+ブループリントは次のようになります。
+
+<!--
+A Blueprint might look something like this:
+ -->
+
+```json
+{
+	"plugins": ["akismet", "gutenberg"],
+	"steps": [
+		{
+			"step": "installTheme",
+			"themeData": {
+				"resource": "wordpress.org/themes",
+				"slug": "twentynineteen"
+			}
+		}
+	],
+	"siteOptions": {
+		"blogname": "My Blog",
```

Comment:
@shimotmk, I think you can translate the blog name, content, and blog description